<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>-20</y>
     <width>1500</width>
     <height>2000</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#mainWidget{
background-color: qlineargradient(spread:pad, x1:0.194, y1:0.198864, x2:0.915423, y2:0.846591, stop:0 rgba(29, 33, 34, 255), stop:1 rgba(31, 231, 146, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>PROCESADOR:0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>PROCESADOR:1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>PROCESADOR:2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>PROCESADOR:3</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="iTable0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>600</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>Previous</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="cache0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>600</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>V</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TAG</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATA</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Caché L1: P0</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="iTable1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>600</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>Previous</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="cache1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>600</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>V</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TAG</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATA</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Caché L1: P1</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="iTable2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>600</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>Previous</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Caché L1: P2</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="cache2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>180</y>
      <width>600</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>V</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TAG</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATA</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="iTable3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>600</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>Previous</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Caché L1: P3</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="cache3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>600</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>V</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TAG</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATA</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>670</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Main Memory</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="memoryTable">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>700</y>
      <width>600</width>
      <height>275</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DATA</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>670</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Purisa&quot;; 
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="directoryTable">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>700</y>
      <width>600</width>
      <height>275</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Meera&quot;;
color: rgb(255,255,255);
background-color: rgb(85, 87, 83);</string>
    </property>
    <column>
     <property name="text">
      <string>ADDRESS</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>STATE</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P1</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P2</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P3</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
